README.md

```diff
@@ -39,24 +41,27 @@ First, you need to mount the gem in the routes file like this
 # config/routes.rb
 
 Rails.application.routes.draw do
-  mount_graphql_devise_for 'User', at: 'api/v1', authenticable_type: Types::CustomUserType, operations: {
-    login: Mutations::Login
-  }, skip: [:sign_up]
+  mount_graphql_devise_for 'User',
+    at: 'api/v1',
+    authenticable_type: Types::MyCustomUserType,
```

Comment:
I changed `CustomUserType` to `MyCustomUserType` because before reading below the example I was assuming that `CustomUserType` was an actual class. I wanted to denote a more "<enter your class name here>". `MyUserType` could also be a good identifier also.

Reply:
The type name change is good but I think for this format we should use parenthesis, WDYT, @00dav00 ? Like this:
```ruby
mount_graphql_devise_for(
  'User',
  at: 'api/v1',
  authenticable_type: Types::MyCustomUserType,
  operations: {
    login: Mutations::Login
  },
  skip: [:sign_up]
)
```

Reply:
+1 for the parenthesis on multiple lines

Reply:
I'll add those.